.gitlab-ci.yml

```diff
@@ -0,0 +1,51 @@
+variables:
+  DOCKER_REPOSITORY: mendersoftware/integration-test-runner
+
+stages:
+  - test
+  - build
+  - publish
+
+include:
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-golang-static.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-docker-build.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-github-status-updates.yml'
+
+test:lint:kubeval:
+  stage: test
+  image: alpine
+  before_script:
+    - wget https://github.com/instrumenta/kubeval/releases/latest/download/kubeval-linux-amd64.tar.gz
```

Comment:
Strangely cumbersome install instruction for something I figured would be a pretty known tool?

Reply:
I just followed their [installation guide](https://www.kubeval.com/installation/)...